auth.cfg

```diff
@@ -0,0 +1,2 @@
+[google]
+key = YOUR_KEY_HERE
```

Comment:
I'm not crazy about adding in conf files into our source tree. What do we do when we install using `pip`? Do we try to distribute these files into standard locations? 

I don't think we really need a conf file here in any case: a CLI argument would seem the simpler choice.